### DISCUSSION TITLE
The Lore of Warcraft

### DISCUSSION THREAD

**Discussion by Yehonal** [chromiecraft/chromiecraft#4182] [Roleplay]
If you are not familiar with the lore behind World of Warcraft and you want to know more about the story up to the Wrath of The Lich King, have a look at this document I have prepared for you!

[The Lore of Warcraft](https://docs.google.com/document/d/1yL0tk60R_jbXWzybJYjXt2Pvu2O-YInFem-inTh_rmg/edit?usp=sharing)

**Comment by pearlhunter** [comment]
Thanks a lot!
